pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "JSON-log-formatter"
+description = 'JSON log formatter'
+authors = [
+    {name = "Marsel Mavletkulov", email = "marselester@ya.ru"},
+]
+readme = "README.rst"
+version = '0.5.2'
+requires-python = ">=3.6"
```

Comment:
Let's remove this requirement if it's possible (this library works with Python 2 as well).

Reply:
Done